libs/login/WordPressLoginFlow/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.android.tools.build:gradle:3.1.0'
```

Comment:
Could you also update the gradle wrapper to `4.4`, and `libs/login/.travis.yml` to use `android-27` and build tools `27.0.3` (not directly related to this PR)? I checked and this is all that's needed for this to build when we push to the subtree.